cmd/libsnap-confine-private/error.h

```diff
@@ -60,6 +60,20 @@ typedef struct sc_error {
  **/
 #define SC_ERRNO_DOMAIN "errno"
 
+/**
+ * Error domain for errors in the libsnap-confine-private library.
+ **/
+#define SC_LIBSNAP_ERROR "libsnap-confine-private"
+
+typedef enum sc_libsnap_error {
+    /** SC_UNSPECIFIED_ERROR indicates an error not worthy of a distinct code. */
```

Comment:
Hm indent did a mix of tabs and spaces here?

Reply:
Sorry, I'll fix that up. I have a rebased patch that fixes formatting but I didn't want to merge it yet so I didn't see the intermediate state that well.

Reply:
Fixed, the bane of mixed formatting where some files use clang and some files use dreaded indent.